sass/themes/default/_theme.scss

```diff
@@ -63,6 +63,7 @@ $tram-color: $livi-tram-green;
 $metro-color: $livi-subway-orange;
 $rail-color: $livi-rail-purple;
 $ferry-color: $livi-ferry-blue;
+$carpool-color: #800000;
```

Comment:
I'd prefer to use the Herrenberg green (which is close to the Fahrgemeinschaft green) as color for carpools. But then we should change the bus symbol background to either purple as in https://de.wikipedia.org/wiki/Datei:BUS-Logo.svg  or better the VVS red https://de.wikipedia.org/wiki/Datei:Bus-Logo_VVS.svg.
For the train, we could use https://de.wikipedia.org/wiki/Datei:S-Bahn-Logo.svg
And metro should be activated as well (some people want to got to Stuttgart where it exists), best would be the U-Bahn blue https://de.wikipedia.org/wiki/Datei:U-Bahnlogo_München.png